test/e2e/scale/scale_test.go

```diff
@@ -200,6 +184,16 @@ var _ = ginkgo.AfterSuite(func() {
 	}
 })
 
+func TestE2E(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	reportDir := os.Getenv(e2eutils.ReportDirEnvVar)
+	if reportDir == "" {
+		reportDir = e2eutils.DefaultReportDir
+	}
+	r := []ginkgo.Reporter{reporters.NewJUnitReporter(path.Join(reportDir, "junit_01.xml"))}
+	ginkgo.RunSpecsWithDefaultAndCustomReporters(t, "AzureDisk CSI Driver End-to-End Tests", r)
+}
+
```

Comment:
This function is needed? We had discussed earlier that it wouldnt be needed right?

Reply:
Oh you are right. I added this by accident. I don't think we need this.

Reply:
I have removed the reporters.